AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/EditHearingTests.cs

```diff
@@ -458,7 +458,7 @@ public async Task Should_remove_endpoint_if_endpoint_is_removed_from_the_endpoin
             ((OkObjectResult)result.Result).StatusCode.Should().Be(200);
 
             _bookingsApiClient.Verify(x => x.RemoveEndPointFromHearingAsync(It.IsAny<Guid>(), It.IsAny<Guid>()), Times.Once);
-            _bookingsApiClient.Verify(x => x.UpdateDisplayNameForEndpointAsync(It.IsAny<Guid>(), It.IsAny<Guid>(), It.IsAny<UpdateEndpointRequest>()), Times.Once);
+            _bookingsApiClient.Verify(x => x.UpdateDisplayNameForEndpointAsync(It.IsAny<Guid>(), It.IsAny<Guid>(), It.IsAny<UpdateEndpointRequest>()), Times.Exactly(2));
```

Comment:
Why should the update be called on a remove?

Reply:
separated the test